### ISSUE TITLE
Gem Bag is useless

### ISSUE THREAD

**Issue by Immortallis1** [Tslat/Advent-Of-Ascension#437]
Gem Bag -> trade five silver coins to get three random lotto gems/tokens. Sounds great until you realize that you can just kill one or two mobs (which tend to drop some small amount of copper coins) to get the same if not more tokens then you just got. Not to mention that tokens in those quantities will rarely get you anything, and are completely useless outside of lotto totems which, while a neat mechanic, are still rather grindy. Gem Bag does absolutely nothing to alleviate that grind.

"Oh but you can get tokens for places that you haven't been!" yeah that's no help if I can't even use them due to lack of a lottoman since I haven't been there. There is no reason whatsoever to spend your silver coins on gem bags instead of saving them for an actually usable material like magic repair dust or shiny boxes or those dimension-specific trades or something.

This may have been somewhat usable when foraging was broken enough to give you lunaver coins so you could buy large amounts (even that's a stretch by the way, if you have lunaver you can just buy better stuff) but now it's just awful and almost, if not every other coin trade is superior. It really should be heavily buffed in some way

**Comment by Cruuk** [comment]
I made a suggestion to buff Gem bag a long while back, and the solution I proposed was that if you use the gem bag in a dimension, you are *much* more likely to get the tokens for said dimension.

It seems you get an average of 2-3 tokens per use with the gem bag. So for example if you used it in Abyss you would almost always get 2-3 Abyss tokens instead, with a... 5-10% chance to get another type of token.

Also (this next one wasn't part of my original suggestion I just came up with this right now), maybe if you have the luck effect you'd have a chance to get another token when using it? As in increasing the max tokens to like 4 instead of 3 or something.
